src/Components/benchmarkapps/Wasm.Performance/dockerfile

```diff
@@ -28,7 +28,7 @@ RUN .dotnet/dotnet publish -c Release --no-restore -o /app ./src/Components/benc
 RUN chmod +x /app/Wasm.Performance.Driver
 
 WORKDIR /app
-FROM selenium/standalone-chrome:103.0 as final
+FROM selenium/standalone-chrome:106.0 as final
```

Comment:
Just curious: No 107 image yet❔

Reply:
Not from what I was able to find on https://hub.docker.com/r/selenium/standalone-chrome/tags.